cognoml/analysis.py

```diff
@@ -56,26 +56,34 @@ def classify(sample_id, mutation_status, **kwargs):
     pipeline.fit(X=X_train, y=y_train)
     #cv_score_df = grid_scores_to_df(clf_grid.grid_scores_)
     
-    obs_df['predicted_status'] = pipeline.predict(X)
-    obs_df['predicted_score'] = pipeline.decision_function(X)
-    #obs_df['predicted_prob'] = pipeline.predict_proba(X)
-        
-    is_testing = obs_df.testing.astype(bool)
-    y_pred_train = obs_df.predicted_score[~is_testing]
-    y_pred_test = obs_df.predicted_score[is_testing]
+    predict_df = pd.DataFrame.from_items([
+        ('sample_id', X_whole.index),
+        ('predicted_status', pipeline.predict(X_whole)),
+        ('predicted_score', pipeline.decision_function(X_whole)),
+        ('predicted_prob', pipeline.predict_proba(X_whole)[:, 1]),
```

Comment:
@yl565 what's the best way to see if a pipeline supports `predict_proba`? We can upgrade to sklearn 18 once that's released, if that will make things easier.